website/pages/en/sps/triggers-example.mdx

```diff
@@ -0,0 +1,107 @@
+---
+title: Example Susbtreams Trigger
+---
+
+Here you’ll walk through a Solana based example for setting up your Substreams-powered subgraph project. If you haven’t already, first check out the [Getting Started Guide](https://github.com/streamingfast/substreams/blob/enol/how-to-guides/docs/new/how-to-guides/intro-how-to-guides.md) for more information on how to initialize your project.
+
+Consider the following example of a Substreams manifest (`substreams.yaml`), a configuration file similar to the `subgraph.yaml`, using the SPL token program Id:
```

Comment:
Capitalizing just the first letter of "Id" seems strange to me, but I could be wrong. I feel like this is a lot more common:
```suggestion
Consider the following example of a Substreams manifest (`substreams.yaml`), a configuration file similar to the `subgraph.yaml`, using the SPL token program ID:
```